core/document/src/document.rs

```diff
@@ -418,6 +418,14 @@ impl Document {
 
 				Some(vec![DocumentResponse::DocumentChanged, DocumentResponse::FolderChanged { path }])
 			}
+			Operation::SetLayerOpacity { path, opacity } => {
+				self.mark_as_dirty(path)?;
+				self.layer_mut(&path).unwrap().opacity = *opacity;
```

Comment:
don't use `unwrap()` if there is a realistic chance the given path is not valid, use `?` instead